edmq_backend/src/scripts/generate_song_list.py

```diff
@@ -0,0 +1,34 @@
+import requests
+import json
+import shazam_data
+
+# Youtube API Request
+part = 'snippet'
+playlist_id = 'PLu-qQLnSm9zQwyCVh_c9uXJ6PDbEFBU8P'
+api_key = 'AIzaSyAJtLJ-uNEpBLZqWHngF_Xpji2NmHFsqSQ'
```

Comment:
Eventually this should be stored into a separate file that is git-ignored because api keys shouldn't be publicly available on github

Reply:
Will generate new API key and fix. I'll do this for the shazam api also.